internal/configs/version1/nginx.ingress.tmpl

```diff
@@ -2,8 +2,8 @@
 {{range $upstream := .Upstreams}}
 upstream {{$upstream.Name}} {
 	{{if ne $upstream.UpstreamZoneSize "0"}}zone {{$upstream.Name}} {{$upstream.UpstreamZoneSize}};{{end}}
-	{{if $upstream.LBMethod }}{{$upstream.LBMethod}};{{end}}
-	{{range $server := $upstream.UpstreamServers}}
+{{if $upstream.LBMethod }}{{$upstream.LBMethod}};{{end}}
+{{range $server := $upstream.UpstreamServers}}
```

Comment:
Why is this indentation being changed? It was correct the way it was (it should be inline with the upstream block opened on line 3)

Reply:
This was caused by the linter in my IDE. Will revert the change.

Reply:
fixed